app/.gitignore

```diff
@@ -1,2 +1,3 @@
 /build
-/src/main/java/com/example/whereto/ParseApplication.java
\ No newline at end of file
+/src/main/java/com/example/whereto/ParseApplication.java
+/src/main/java/com/example/whereto/Constants.java
```

Comment:
Could you help me understand why there are two gitignores? and why the constants are included in it?

Reply:
I am not sure why there are 2 gitignores, I believe one is for the project and one is for the app.
I included my API key in the Constants file so I added it to the gitignore. I can move it out and make another file if that is better though!